build.gradle

```diff
@@ -259,6 +261,19 @@ project('client') {
         testCompile group: 'io.netty', name: 'netty-transport-native-epoll', version: nettyVersion, classifier: 'linux-x86_64'
     }
 
+    tasks.withType(JavaCompile) {
+        options.compilerArgs << "--release" << clientJavaVersion
+    }
+
+    // If the user has the client to build with a different version and there exists an appropriate JDK,
```

Comment:
Are we adding this block in order to run tests of client on jdk8 ?
I think this is not really needed. 
If we set --release = 8 in the client and in all dependencies it should be enough.

It is better to have some external project / integration tests suite that uses the client.
Having here such support looks to me overwhelming

Reply:
This functionality was introduced in response to your comment here: https://github.com/pravega/pravega/pull/5121#discussion_r482239853. I'm happy with removing it if you think it is overkill.

Reply:
@co-jo yes, to me is overkill. Let's remove this complexity.